intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -231,7 +231,11 @@ \subsection{Your data directory (\$VSC\_DATA)\label{subsec:data-directory}}
 \ifgent
 If you are running out of quota on your \$VSC\_DATA filesystem you can
 request a VO (Virtual organisation). VO members get an extra DATA filesystem: \$VSC\_DATA\_VO)
```

Comment:
`An extra directory on the DATA filesystem is created for a VO: \$VSC\_DATA\_VO`

Reply:
we should be consistent about how we name things yes. In an upcoming data-policy we are currently opting for 'data partition' or 'partition' instead of filesystem/fileset/storage system. But that's a different issue to be handled separately from this PR

Reply:
I wasn't really pointing out an inconsistency, the `extra DATA filesystem` was simply incorrect, all they get is a directory on an existing filesystem, we don't create a new filesystem for them.

Technically, I'm not sure 'partition' is 100% correct. Filesystem is though, and it's a fairly common term, just like 'directory', while partition is less common and may raise questions (but maybe I'm biased, I haven't used Windows for a long time).
(What are the `D:`, `E:` additional locations next to `C:` called in Windows?)

That they're actually getting a fileset on a filesystem is irrelevant to the users, all they see and need to care about is a directory...

Reply:
@JensTimmerman This line should change too, `an extra DATA filesystem` is just wrong imho...

Also, please sync your branch with current `master` and run the `check-version.sh` to bump the version.

Reply:
again, see #158

Reply:
@boegel why don't the tests do this? They merge with master, no?